ETC.tsv

```diff
@@ -12393,9 +12393,9 @@ ETC_20150714_012394	Move
 ETC_20150714_012395	Don't move
 ETC_20150714_012396	Impure energy is about to pour over! Run away!
 ETC_20150714_012397	You've defeated Gaigalas!{nl}Cut down the root of Bramble
-ETC_20150714_012398	You've defeated Molek!{nl}Cut down the root of Bramble
+ETC_20150714_012398	$You've defeated Molich!{nl}Cut down the root of Bramble
```

Comment:
Change it to: $You have defeated Molich!{nl}Cut down the root of Bramble

I have this in my PR as well, same with line 012397 (Gaigalas), I added a $ for the sake of not getting merge conflicts. :)